semantics/common/configuration.k

```diff
@@ -101,6 +101,18 @@ module COMMON-CONFIGURATION
                          <access-specifier> .K </access-specifier>
                     </class>
                </classes>
+
+               <cpp-enums>
+                    <cppenum multiplicity="*" type="Map">
+                         <enum-id> 0 </enum-id>
```

Comment:
ah, yes. This is sort of an ongoing issue. There is actually a bug in the semantics dealing with anonymous enums and classes at the moment. I would prefer to solve that issue separately from the issue from the enums, but if you want to tackle it now, what I would suggest is to give all anonymous classes and enums a numeric id as part of their CId. Like, right now ClangKast.cc parses the name of an anonymous struct as `#NoName`. I would propose to change that to `#NoName(Int)`. But it has to happen in the parser before the semantics even starts or the code will not behave correctly no matter what you do.